apps/etterlatte-saksbehandling-ui/client/src/components/person/aktivitet/vurderingAvAktivitetsplikt/components/AktivitetsgradReadMore.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Box, ReadMore } from '@navikt/ds-react'
+
+export const AktivitetsgradReadMore = () => {
```

Comment:
Skal denne brukes flere steder? Hvis ikke blir det imo litt unødvendig å splitte den ut i egen komponent da det tilføyer ekstra kompleksitet som ikke trengs.

Samme gjelder for unntakreadmore